source/community/install-from-source.html.md

```diff
@@ -57,7 +57,7 @@ Details on installing an image using a quickstart file are available from
    1. Initialize the database:
 
       ```bash
-      service postgresql initdb
+      postgresql-setup initdb
```

Comment:
I don't know what's the difference is but I believe `systemctl postgresql initdb` would be "closer" to what we had before (sysvinit vs. systemd)